xunit-autolabeler-v2/ast_parser/core/polyglot_parser.py

```diff
@@ -65,6 +69,34 @@ def _recursor(method: pdd.PolyglotDriftData) -> pdd.PolyglotDriftData:
         method.region_tags = list(set(method.region_tags))
         method.test_methods = list(set(method.test_methods))
 
+    """
+    EDGE CASE:
+        Some snippets (e.g. Python jobs API) are wrapped in a separate
+        "snippet invocation method" (e.g. "run_sample"), that itself is
+        invoked within the tests.
+
+    SOLUTION:
+        Move snippet-invocation method tests to their invoked [child] methods
+        (This *cannot* be done with the child-labeling system, as that looks
+         for tests in the child methods - *not* the parent ones!)
+    """
+    for method in source_methods:
+        if method.name in constants.SNIPPET_INVOCATION_METHODS:
```

Comment:
Skip the item to reduce indentation:

if method.name not in constants.SNIPPET_INVOCATION_METHODS:
  continue

Reply:
Good catch; fixed.